style.css

```diff
@@ -0,0 +1,60 @@
+*
```

Comment:
Just some advice: you might want to have `margin: 0`, `padding: 0` and `box-sizing: border-box` here.